languages.ts

```diff
@@ -423,7 +423,7 @@ export const languageSpecs: LanguageSpec[] = [
             languageID: 'clojure',
             fileExts: ['clj', 'cljs', 'cljx'],
             commentStyle: lispStyle,
-            identCharPattern: /[A-Za-z0-9_\-!?]/,
+            identCharPattern: /[A-Za-z0-9_\-!?+*>]/,
```

Comment:
```suggestion
            identCharPattern: /[A-Za-z0-9_\-!?+*<>]/,
```

Reply:
Ah, looking at the official list now, I see they've expanded the permitted options (should have checked the source, not just the StackOverflow...). I'll update